packages/DynamicContentLoader/src/FacetsUpdater.js

```diff
@@ -82,7 +83,12 @@ export default class FacetsUpdater extends HTMLElement {
     }
 
     #getRequestConfig({ searchParams }) {
-        return { url: `${this.#getEndpoint()}?${searchParams.toString()}` };
```

Comment:
@andimeier-ch This always appended a `?` to the URL, no matter if a query string was present or not. That lead to issues with the backend, sometimes.